docs/pages/play.vue

```diff
@@ -154,27 +144,17 @@
                 </b-btn>
               </div>
 
-              <codemirror
-                v-model="js"
-                mode="javascript"
-              ></codemirror>
+              <codemirror v-model="js" mode="javascript"></codemirror>
             </b-card>
           </b-col>
         </transition-group>
       </b-col>
 
       <!-- Right/bottom column -->
-      <b-col
-        key="B"
-        :md="vertical || full ? 12 : 6"
-        sm="12"
-      >
+      <b-col key="B" :md="vertical || full ? 12 : 6" sm="12">
         <b-row>
           <!-- Result column -->
-          <b-col
-            cols="12"
-            class="mt-3"
-          >
+          <b-col cols="12" class="mt-3">
```

Comment:
you can just use prop `col` instead of `cols="12"`

Reply:
I think...

Reply:
Actually... no... since there is another col adjacent this one

Reply:
Doesn't seem to work...

Reply:
Reverted...